### PR TITLE
Check localStorage is available before using it

### PR DESCRIPTION
**Pull request by schwers**

### CODE REVIEW

**Comment by schwers** [comment]
:eyeglasses: @ajacksified @curioussavage (before travis finished, but checked locally)

File: src/lib/localStorageAvailable.es6.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const FEATURE_TEST_LOCAL_STORAGE = 'mobile_reddit_local_storage_feature_test';
+
+export default function localStorageAvailable() {
+  try {
+    if (typeof localStorage !== 'undefined') {
+      localStorage.setItem(FEATURE_TEST_LOCAL_STORAGE, 'yes');
```
Comment (ajacksified) [inline src/lib/localStorageAvailable.es6.js]: please make 'yes' a constant too

Comment (ajacksified) [inline src/lib/localStorageAvailable.es6.js]: `const YES = 'yes';`

**Comment by ajacksified** [comment]
:fish:

**Pull request merged**